src/summary/documentation/documentation-section.component.tsx

```diff
@@ -6,12 +6,15 @@ import VitalsCard from "./vitals-card.component";
 
 export default function DocumentationSection(props: DocumentationSectionProps) {
   return (
-    <>
+    <div>
```

Comment:
By leaving this as <>, it wasn't picking up the styling and the two cards remained side by side.